rfqmsg/sell_request.go

```diff
@@ -152,12 +152,10 @@ func NewSellRequestFromWire(wireMsg WireMessage,
 
 // Validate ensures that the quote request is valid.
 func (q *SellRequest) Validate() error {
-	// Ensure that the asset specifier is set.
-	//
-	// TODO(ffranr): For now, the asset ID must be set. We do not currently
-	//  support group keys.
-	if !q.AssetSpecifier.HasId() {
-		return fmt.Errorf("asset id not specified in SellRequest")
+	// Ensure that the asset specifier has either an asset ID or a group
+	// key set.
+	if !q.AssetSpecifier.HasId() && !q.AssetSpecifier.HasGroupPubKey() {
```

Comment:
Maybe add `IsEmpty()` method to specifier? Or `AssertNotEmpty()` that then also returns the error that we can just return here?